ADAL/tests/integration/legacytest/ADLegacyMacTokenCache.m

```diff
@@ -140,7 +140,12 @@ - (nullable NSData *)serialize
     NSData *result = nil;
     @try
     {
-        result = [NSKeyedArchiver archivedDataWithRootObject:wrapper];
+        if (@available(macOS 10.13, iOS 12.0, *)) {
```

Comment:
same here - purpose of this class is to have test with legacy implementation compatibility. Should we keep it as is for test purposes?

Reply:
Since we are using iOS 11 as the deployment target, the test cases will not show a deprecated warning. I have reverted changes to legacy test cases in the next commit(cc40a79). Thanks